### DISCUSSION TITLE
windows系统 能支持调用 cuda吗？

### DISCUSSION THREAD

**Discussion by brucetiancai** [mlc-ai/mlc-zh#48] [Q&A]
请教下，目前我是在windows下装的anaconda，
但是看了下 https://mlc.ai/wheels 网址下，cuda对应的都是Linux的
是不是想用cuda，必须得换成Linux呢？

**Comment by Hzfengsy** [comment]
如果想使用cuda 建议使用Linux，如果需要在Windows下使用可以基于 https://github.com/mlc-ai/relax 手动编译

**Comment by learning-and-learning1651880** [comment]
可以用wsl来安装
